pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -837,6 +837,13 @@ private void extractEndpointBlobConfig() {
       String iotConfig = extractConfigBlob(IOT_ENDPOINT_CONFIG.value());
       extractedEndpoint = iotConfig == null ? null
           : OBJECT_MAPPER.readValue(iotConfig, EndpointConfiguration.class);
+      if (extractedEndpoint != null) {
+        // TODO: Refactor extractConfigBlob() to get any blob meta parameters like nonce.
+        if (deviceConfig.blobset.blobs.get(IOT_ENDPOINT_CONFIG.value()) != null) {
```

Comment:
nit: there's a .containsKey() method that's preferred (according to "them") over the .get() != null check...